sale_order_line_packaging_qty/models/sale_order_line.py

```diff
@@ -86,5 +87,69 @@ def _onchange_product_uom_qty(self):
         # TODO Drop once https://github.com/odoo/odoo/pull/49150/ is merged
         res = super()._onchange_product_uom_qty()
         if not res:
-            res = self._check_package()
+            res = self._check_package() or self._check_qty_is_pack_multiple()
         return res
+
+    @api.onchange("product_id")
+    def product_id_change(self):
+        res = super().product_id_change()
+        if self.product_id.only_sell_by_pack:
+            self.product_uom_qty = min(self.product_id.packaging_ids.mapped("qty"))
+        return res
+
+    def _check_qty_is_pack_multiple(self):
+        """ Check only for product with only_sell_by_pack
+        """
+        # self.product_uom_qty != 1 cause is the defaut value
```

Comment:
Is this a todo that should be implemented ?

Reply:
yes should be remove it's was comment for me in the dev phase sorry for the noise